src/woo_publications/metadata/api/filters.py

```diff
@@ -0,0 +1,29 @@
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from woo_publications.api.filters import URLFilter
+
+from ..models import InformationCategory
+
+
+class InformationCategoryFilterset(FilterSet):
```

Comment:
```suggestion
class InformationCategoryFilterSet(FilterSet):
```
please use consistent casing